apps/nextra/components/landing/sections/ToolingSection.tsx

```diff
@@ -0,0 +1,115 @@
+import { i18nConfig } from "@docs-config";
+import Image from "next/image";
+import { Link } from "nextra-theme-docs";
+import { useRouter } from "nextra/hooks";
+import { ReactNode } from "react";
+import { IconArrowRight } from "../components/Icons";
+import { Section, SectionHeader } from "../components/Section";
+
+export function ToolingSection() {
+  const { locale } = useRouter();
+  const t = i18nConfig[locale!];
+
+  return (
+    <Section>
+      <SectionHeader>{t.toolingSectionHeadline}</SectionHeader>
+      <div className="flex flex-col lg:flex-row justify-center items-center lg:items-stretch w-full">
+        <ToolingCard
+          illustration={
+            <>
+              <Image
+                className="block dark:hidden"
+                src="/landing/indexer-illustration-light.svg"
+                alt={t.indexerIllustrationAlt}
+                fill
+              />
+              <Image
+                className="hidden dark:block"
+                src="/landing/indexer-illustration-dark.svg"
+                alt={t.indexerIllustrationAlt}
+                fill
+              />
+            </>
+          }
+          logo={
+            <Image
+              src="/landing/graphql-logo.svg"
+              alt={t.graphqlLogoAlt}
+              width={56}
+              height={65}
+            />
+          }
+          label={t.indexerLabel}
+          description={t.indexerDescription}
+          href="" // TODO: Add link to relevant docs page
+        />
+        <ToolingCard
+          illustration={
+            <>
+              <Image
+                className="block dark:hidden"
+                src="/landing/sdk-illustration-light.svg"
+                alt={t.sdkIllustrationAlt}
+                fill
+              />
+              <Image
+                className="hidden dark:block"
+                src="/landing/sdk-illustration-dark.svg"
+                alt={t.sdkIllustrationAlt}
+                fill
+              />
+            </>
+          }
+          logo={
+            <Image
+              src="/landing/typescript-logo.svg"
+              alt={t.typescriptLogoAlt}
+              width={60}
+              height={59}
+            />
+          }
+          label={t.sdkLabel}
+          description={t.sdkDescription}
+          href="" // TODO: Add link to relevant docs page
+        />
+      </div>
+    </Section>
+  );
+}
+
+interface ToolingCardProps {
+  illustration: ReactNode;
+  logo: ReactNode;
+  label: string;
+  description: string;
+  href: string;
+}
+
+function ToolingCard(props: ToolingCardProps) {
+  return (
+    <Link
```

Comment:
Could we add `cursor: "pointer"`?

Reply:
Since `Link` renders an `a` tag, we don't actually need to add `cursor: "pointer"`  here. The reason why the UI currently uses the default cursor style is because the `href`s are empty right now. As soon as we fill in the `href`, the cursor will display as a pointer.